src/local_client_base.py

```diff
@@ -50,13 +50,16 @@ def __init__(self, update_statuses):
         self.database_parser = None
         self.config_parser = None
         self.uninstaller = None
+        self.load_local_files_done = False
         self.installed_games_cache = self.get_installed_games()
 
         loop = asyncio.get_event_loop()
         loop.create_task(self._register_local_data_watcher())
         loop.create_task(self._register_classic_games_updater())
         self.classic_games_parsing_task = None
 
+
+
```

Comment:
according to pep8 every method should be separated by only one empty line https://www.python.org/dev/peps/pep-0008/#blank-lines

Reply:
agreed